Library/src/main/java/com/pusher/android/notifications/PushNotificationRegistration.java

```diff
@@ -59,14 +59,20 @@ public PushNotificationRegistration(
         this.manifestValidator = manifestValidator;
     }
 
+    /**
+     * @deprecated this feature is being decommissioned, please use Pusher Beams instead
```

Comment:
How about adding a link to Beams' home page?

Reply:
I was thinking about it. I was also wondering about the migration guide, but I was worried that it's going to go away in a few months and no one will update the library to fix a dead-link.

Reply:
I think that the link to the migration guide is a good one. I wouldn't worry about the dead link.

I also think that such pages should remain online, even if the content is replaced with "This is an old guide for a discontinued product."